package/network/utils/ltq-dsl-base-files/Makefile

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (C) 2019 OpenWrt.org
```

Comment:
You are not affiliated with OpenWrt, so AFAIK you cannot put an OpenWrt copyright here.

Reply:
OK, I'll remove that line.

Reply:
Either put your name there or just omit the copyright and just keep the license.

Reply:
Done